rpc-server/src/storage.rs

```diff
@@ -290,4 +298,45 @@ impl ScyllaDBManager {
             &data_value,
         )?)
     }
+
+    pub async fn get_block_by_height_and_shard_id(
+        &self,
+        block_height: near_primitives::types::BlockHeight,
+        shard_id: near_primitives::types::ShardId,
+    ) -> anyhow::Result<(u64, u64)> {
+        let rows = Self::execute_prepared_query(
+            &self.scylla_session,
+            &self.get_height_included_and_shard_id_by_block_height,
+            (num_bigint::BigInt::from(block_height),),
+        )
+        .await?
+        .rows()?;
+
+        let height_included_and_shard_id = rows
+            .into_typed::<(num_bigint::BigInt, num_bigint::BigInt)>()
```

Comment:
The code would be much cleaner if we introduce custom type with from()/try_from() method